src/templates/search.pug

```diff
@@ -10,7 +10,7 @@ block content
         input.input.is-rounded.searchbar(
           type="search"
           placeholder="Search the HNS Blockchain"
-          maxlength="63")
+          maxlength="64")
```

Comment:
we switched this to 63 to better support names, but of course that broke txs. Our Search now better supports multiple results, and properly routes 64 character searches to transactions, so it's safe to restore this to 64